api/src/main/kotlin/handler/CoursebookHandler.kt

```diff
@@ -0,0 +1,36 @@
+package com.wafflestudio.snu4t.handler
+
+import com.wafflestudio.snu4t.common.enum.Semester
+import com.wafflestudio.snu4t.coursebook.data.CoursebookOfficialResponse
+import com.wafflestudio.snu4t.coursebook.data.toResponse
+import com.wafflestudio.snu4t.coursebook.service.CoursebookService
+import com.wafflestudio.snu4t.middleware.SnuttRestApiNoAuthMiddleware
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.server.ServerRequest
+import org.springframework.web.reactive.function.server.ServerResponse
+
+@Component
+class CoursebookHandler(
+    private val coursebookService: CoursebookService,
+    snuttRestApiNoAuthMiddleware: SnuttRestApiNoAuthMiddleware
+) : ServiceHandler(
+    handlerMiddleware = snuttRestApiNoAuthMiddleware
+) {
+    suspend fun getCoursebooks(req: ServerRequest): ServerResponse = handle(req) {
+        coursebookService.getCoursebooks().map { it.toResponse() }
+    }
+
+    suspend fun getLatestCoursebook(req: ServerRequest): ServerResponse = handle(req) {
+        coursebookService.getLatestCoursebook().toResponse()
+    }
```

Comment:
프로젝트 내에서 toResponse등의 패턴을 안 쓰고 있는데요
handler안 메소드는 함수 body 타입이 감춰져있어서 
실제 body 타입을 명시해주는 게 좋을 것 같아요

프로젝트 내에서 생성자, 생성자 확장 함수, from 패턴 다양하게 쓰고 있는데 생성자 확장 함수 형태가 제일 흔해서 
아래와 같은 방식으로 만들면 좋을 것 같아요

```suggestion
    suspend fun getCoursebooks(req: ServerRequest): ServerResponse = handle(req) {
        coursebookService.getCoursebooks().map { CoursebookResponse(it) }
    }

    suspend fun getLatestCoursebook(req: ServerRequest): ServerResponse = handle(req) {
        val latestCoursebook = coursebookService.getLatestCoursebook()
        CoursebookResponse(latestCoursebook)
    }
```